widgetssdk/build.gradle

```diff
@@ -64,9 +64,7 @@ dependencies {
     // debugImplementation "com.squareup.leakcanary:leakcanary-android:$leakCanaryVersion"
 
     // Replaced with a local AAR file
-//    implementation "com.glia:android-sdk:$gliaSdkVersion"
-    // Glia sdk aar
-    implementation(name: "glia-core-sdk-$gliaSdkVersion", ext: 'aar')
+    implementation "com.glia:android-sdk:$gliaSdkVersion"
```

Comment:
Is fixed in develop will go to master with 1.6 release

Reply:
Done, I've changed the base for this PR and now it looks into develop branch.

Reply:
Did the rebase instead of merge :) do `git pull -r` in order to get the updates

atm. unneeded deps from master got in - had to clear them, now it should be fine @dukhovnyi